src/base/utils/string.h

```diff
@@ -49,6 +47,14 @@ namespace Utils
         bool naturalCompareCaseInsensitive(const QString &left, const QString &right);
 
         QString wildcardToRegex(const QString &pattern);
+
+        template <typename T>
+        T unquote(const T &str)
+        {
+            if (str.startsWith('\"') && str.endsWith('\"'))
```

Comment:
Ah, OK. Perhaps I did not refresh browser tab and did not see the comment in the code.